packages/ubuntu_desktop_installer/lib/l10n/app_es.arb

```diff
@@ -0,0 +1,113 @@
+  "otherOptions": "Otras opciones",
+  "installThirdPartyTitle": "Instalar software de terceros para controladores de gráficos y WiFi, así como formatos de contenido adicionales",
+  "installThirdPartySubtitle": "Este software está sujeto a términos de licencia incluidos en su documentación. Algunos son propietarios.",
+
+  "allocateDiskSpace": "Asignar espacio de disco",
+  "startInstallingButtonText": "Comenzar la instalación",
+  "diskHeadersDevice": "Dispositivo",
+  "diskHeadersType": "Tipo",
+  "diskHeadersMountPoint": "Punto de montaje",
+  "diskHeadersSize": "Tamaño",
+  "diskHeadersUsed": "Usado",
+  "diskHeadersSystem": "Sistema",
+  "diskHeadersFormat": "Formato",
+
+  "writeChangesToDisk": "Escribir cambios en el disco",
+  "writeChangesFallbackSerial": "disco",
+  "@writeChangesFallbackSerial": { "description": "Nombre predeterminado que mostrar para un disco sin uno de serie (poco probable)" },
+  "writeChangesDescription": "Si continúa, los cambios listados más abajo se escribirán en los discos. Podrá hacer más cambios manualmente.",
+  "writeChangesPartitionTablesHeader": "Las tablas de partición de los siguientes dispositivos se modifican:",
+  "writeChangesPartitionTablesEntry": "{serial} ({path})",
+  "@writeChangesPartitionTablesEntry": {
+    "description": "Entrada de un disco cuya tabla de partición está siendo modificada",
+    "placeholders": {
+      "serial": {},
+      "path": {}
+    }
+  },
+  "writeChangesPartitionsHeader": "Las particiones siguientes se van a formatear:",
+  "writeChangesPartitionEntryPrimaryFull": "partición #{partitionNumber} de {diskSerial} ({diskPath}) como {fstype} usada para {mountPath}",
+  "@writeChangesPartitionEntryPrimaryFull": {
+    "description": "Entrada para una partición primaria sin particiones secundarias",
+    "placeholders": {
+      "partitionNumber": {},
+      "diskSerial": {},
+      "diskPath": {},
+      "fstype": {},
+      "mountPath": {}
+    }
+  },
+  "writeChangesPartitionEntryPrimary": "partición #{partitionNumber} de {diskSerial} ({diskPath}) como",
+  "@writeChangesPartitionEntryPrimary": {
+    "description": "Entrada para una partición primaria con particiones secundarias",
+    "placeholders": {
+      "partitionNumber": {},
+      "diskSerial": {},
+      "diskPath": {}
+    }
+  },
+  "writeChangesPartitionEntrySecondary": "        partición # como {fstype} usada para {mountPath}",
+  "@writeChangesPartitionEntrySecondary": {
+    "description": "Entrada para una partición secundaria",
+    "placeholders": {
+      "fstype": {},
+      "mountPath": {}
+    }
+  },
+
+  "chooseYourLookPageTitle": "Seleccione el aspecto",
+  "chooseYourLookPageHeader": "Puede cambiarlo después en la configuración de apariencia.",
+  "chooseYourLookPageDarkSetting": "Oscuro",
+  "chooseYourLookPageLightSetting": "Luminoso",
+  "chooseYourLookPageLightBodyText": "Todo es luminoso y brillante",
+  "chooseYourLookPageDarkBodyText": "La oscuridad es mi vieja amiga"
```

Comment:
This shouldn't be translated literally (the English sentence is an excerpt from Simon and Garfunkel's *The Sound of Silence*, a literal translation would at best be confusing), rather replaced with an idiomatic equivalent, or a default, neutral description (like I did for the French translation).

Reply:
I know that song it's just I didn't find any equivalents in spanish but I'm open to suggestions.

Reply:
I mean the fact that they included that excerpt it's a joke in itself so I thought a little humor won't hurt. Instead of using a more plain description I decided to use not a transliteration but a similar catchy sentence which is not too informal but not too formal either since I think that was the original idea of using that excerpt. I mean, everyone understands what dark and light means. No need for clarification, no room for confusion (but it's just my opinion).

Reply:
I agree, and I think the English sentence should be changed to be more neutral, or the text label be removed altogether as it doesn't add any valuable information. Until we get a design opinion on this, I'm fine with leaving your translation as is.